Fix missing awesome-pages plugin

### Overview
We need to fix GHA. Github Actions to deploy docs has been broken since [bf128f5](https://github.com/hackforla/peopledepot/commit/bf128f5d4bdd3f04b6d8e01ef409421151c948c6). Here's the message

```console
[1](https://github.com/hackforla/peopledepot/actions/runs/7380914738/job/20078846783#step:7:1)
Run mkdocs gh-deploy --force
ERROR   -  Config value 'plugins': The "awesome-pages" plugin is not installed

Aborted with 1 configuration errors!
Error: Process completed with exit code 1.
```

### Action Items

- [ ] fix the gha

--- a/.github/workflows/deploy-docs.yml
+++ b/.github/workflows/deploy-docs.yml
@@ -22,5 +22,6 @@ jobs:
             mkdocs-material-
       - run: pip install \
           mkdocs-material==9.1.17 \
+          mkdocs-awesome-pages-plugin==2.9.1 \
           mkdocs-autolinks-plugin==0.7.1
       - run: mkdocs gh-deploy --force

